### PR TITLE
Footer and Navigation Feature

### PR DESCRIPTION
**Pull request by Merdoth**
#### What does this PR do?
-Application Navigation
-Application Footer

#### Description of Task to be completed?
- Shows the application landing page and the initial screen you see when you visit the web application with inclusion of page footer and navigation

#### How should this be manually tested?
- run the application using `npm run dev` visit `http://localhost:3000`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: Feature
- story title: Navigation and Footer

#### Screenshots (if appropriate)
N/A

#### Question
N/A

### CODE REVIEW

File: components/Common/Footer.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+/* eslint react/jsx-filename-extension: 0 */
+/* eslint react/jsx-no-literals: 0 */
+/* eslint arrow-body-style: 0 */
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: remove these eslint disable

File: components/Common/Footer.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+const {
+    NUMBER,
+} = HOME_STRINGS;
+
+const Footer = () => {
+    return (
+        <React.Fragment>
+            <Head>
+                <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossOrigin="anonymous" />
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: save this `href` value as a string in your `constants.js`

File: components/Common/common.scss [hunk 3]
```diff
@@ -0,0 +1,205 @@
+html, body {
```
Comment (Way2nnadi) [inline components/Common/common.scss:1]: go through this file and fix the formatting

File: components/Common/NavigationBar.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+/* eslint react/jsx-filename-extension: 0 */
+/* eslint react/jsx-no-literals: 0 */
+/* eslint arrow-body-style: 0 */
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
```
Comment (Way2nnadi) [inline components/Common/NavigationBar.js]: remove these eslint disable

File: components/Common/NavigationBar.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
```
Comment (Way2nnadi) [inline components/Common/NavigationBar.js:2]: go through this file and save all strings that is not a `className` to the `constants.js` file

File: components/Common/Footer.js [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+/* eslint react/jsx-filename-extension: 0 */
+/* eslint react/jsx-no-literals: 0 */
+/* eslint arrow-body-style: 0 */
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
+
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: go through this file and save all strings that is not a `className` to the `constants.js` file

File: components/Common/Footer.js [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                        <p className="footer1-text2-header">Useful Links</p>
+                        <div className="useful-links-wrapper">
+                            <ul className="useful-links">
+                                <li>
+                                    <a>Testimonials</a>
+                                </li>
+                                <li><a>FAQ</a></li>
+                                <li><a>Terms & Conditions</a></li>
+                                <li><a>Our Partners</a></li>
+                                <li><a>Services</a></li>
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: save these strings in an array and map over them to create this section of the component
for example:
```
const LIST_OF_STRINGS = ['hello', 'world']
...
<ul>
    LIST_OF_STRINGS.map(string => (
        <li><a>{string}</a></li>
    ))
</ul>
```

File: components/Common/Footer.js [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+                            </ul>
+                            <ul className="useful-links links2">
+                                <li><a>Free Services</a></li>
+                                <li><a>About us</a></li>
+                                <li><a>News</a></li>
+                                <li><a>Contact</a></li>
+                                <li><a>Feedback</a></li>
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: here as well map over a list of strings

File: components/Common/Footer.js [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+                            {' '}
+                            <a>LifeAID</a>
+                        </p>
+
+                    </div>
+                    <div className="icons">
+                        <i className="fab fa-instagram" />
+                        <i className="fab fa-facebook-f" />
+                        <i className="fab fa-twitter" />
+                        <i className="fab fa-linkedin-in" />
```
Comment (Way2nnadi) [inline components/Common/Footer.js]: map over a list of classNames to create these `i` tags

File: components/Common/NavigationBar.js [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+                    <li>
+                        <Link href="#contact">
+                            <a className="listitem">Contact</a>
+                        </Link>
+                    </li>
+                    <li>
+                        <Link href="/login">
+                            <a className="listitem">Login</a>
+                        </Link>
+                    </li>
```
Comment (Way2nnadi) [inline components/Common/NavigationBar.js]: Same mapping pattern here.

File: components/Common/Footer.js [hunk 11]
```diff
@@ -1,123 +1,121 @@
...
-
+/* eslint-disable react/jsx-indent-props */
 import React from 'react';
 import Head from 'next/head';
 import './common.scss';
-import { HOME_STRINGS } from '../constants';
+import {
+    HOME_STRINGS, ICONS,
+    FOOTER_LINK, FOOTER_LINK2
+}
```
Comment (Way2nnadi) [inline components/Common/Footer.js:11]: all variable imports on one line or on there own line
```
import { HOME_STRINGS, ICONS, FOOTER_LINK, FOOTER_LINK2 } from '../constants';
```
or
```
import {
    HOME_STRINGS,
    ICONS,
    FOOTER_LINK,
    FOOTER_LINK2,
} from '../constants';
```

File: next.config.js [hunk 12]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```
Comment (Way2nnadi) [inline next.config.js]: remove the /* eslint-disable */ in this page. And fix the indents on this file, we use 4 spaces or 2 tabs. You are doing more than that here

File: components/Common/Footer.js (hunk 11 above)
Comment (Merdoth) [reply components/Common/Footer.js:11]: Resolved

File: next.config.js [hunk 13]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-vars */
+const path = require('path');
+const glob = require('glob');
+
+module.exports = {
+    // eslint-disable-next-line no-unused-vars
+    webpack: (config, { dev }) => {
```
Comment (Way2nnadi) [inline next.config.js]: why are you destructuring `dev` if you are not using it. remove the eslint disable on line 2 and 7

Comment (Merdoth) [reply next.config.js]: Resolved

**Review by Way2nnadi — APPROVED** [review]

**Review by codemutatio — APPROVED** [review]

**Pull request merged**